Tc issue 122

vsperez opened the pull request:
I haven't test it jet.

vsperez committed:
Merge pull request #2 from TheTransitClock/develop

Syn dev

vsperez committed:
Fix issue 122

Add param assigned boolean, defalt false i order to return assigned
vehicles only.

It needs to be tested.

vsperez committed:
Change serialization to be able to handle Doube insted of double for
distanceToNextStop,distanceOfNextStopFromTripStart and distanceAlongTrip

nselikoff commented on transitclockApi/src/main/java/org/transitclock/api/rootResources/TransitimeApi.java:
```diff
@@ -306,7 +306,12 @@ public Response getVehiclesDetails(@BeanParam StandardParameters stdParameters,
-			@DefaultValue("3") int numberPredictions) throws WebApplicationException {
+			@DefaultValue("3") int numberPredictions
+			,
+			@Parameter(description=" Return only assigned vehicles",required=false)@QueryParam(value = "assgined") 
```
@vsperez small typo here: "assgined"

nselikoff commented:
Related issue: https://github.com/TheTransitClock/transitime/issues/122

vsperez committed:
Correction of typo.

As suggested by Nathan, the name of parameter assigned was changed to
onlyAssigned

scrudden commented:
@vsperez Have you run any tests on this change? By that I mean have you set up an instance with this change and checked that all the maps work as expected?

vsperez commented:
Sean,

I haven’t tested the last change. I don’t recall  exactly what test I did, so I will do it again and I will let you know.

Best Regards
Vicente

> On May 13, 2019, at 11:54, Sean Óg Crudden <notifications@github.com> wrote:
> 
> @vsperez <https://github.com/vsperez> Have you run any tests on this change? By that I mean have you set up an instance with this change and checked that all the maps work as expected?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/TheTransitClock/transitime/pull/124?email_source=notifications&email_token=AJUAAGFC2NHMHVVI5F6HDXLPVGFMBA5CNFSM4GTFK5EKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODVIYEPY#issuecomment-491881023>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AJUAAGFXZN4MZTSXDEPFTF3PVGFMBANCNFSM4GTFK5EA>.
>

vsperez commented:
Hi Sean,

I was testing the branch, and I could make it work property, but I had some issues, that I am quite sure were solved. I think I am getting crazy!!!! (tc_issue_131 merged on march). I don’t see those changes in develop.

1.- TransitimeApi — getRouteDetails  missing the r parameter.  
2.- IpcRoute  — createStops using if( tripPattern.getDirectionId().compareTo(currentDirectionId)==0) instead of if( tripPattern.getDirectionId()==null || tripPattern.getDirectionId().compareTo(currentDirectionId)==0) (line 225 and 236)
3.- Some improvements in Synoptic view. By the way, with this agency I found some bugs.

I am testing with Intercity…But no buses right now :(. So it is working both: Map and Synoptic but without buses so far.


Regards.
Vicente 

> On May 13, 2019, at 11:54, Sean Óg Crudden <notifications@github.com> wrote:
> 
> @vsperez <https://github.com/vsperez> Have you run any tests on this change? By that I mean have you set up an instance with this change and checked that all the maps work as expected?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/TheTransitClock/transitime/pull/124?email_source=notifications&email_token=AJUAAGFC2NHMHVVI5F6HDXLPVGFMBA5CNFSM4GTFK5EKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODVIYEPY#issuecomment-491881023>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AJUAAGFXZN4MZTSXDEPFTF3PVGFMBANCNFSM4GTFK5EA>.
>

vsperez committed:
Commit of changes for this branch to work. Before merge against develop

vsperez committed:
Merge branch 'develop' of https://github.com/TheTransitClock/transitime
into TheTransitClock-develop

# Conflicts:
#	transitclock/src/main/java/org/transitclock/ipc/data/IpcRoute.java
#	transitclockApi/src/main/java/org/transitclock/api/rootResources/TransitimeApi.java

vsperez committed:
Working with InterCity

vsperez committed:
Error in last commit

scrudden commented:
Deprecated by pull request #142
